reputation/test_reputation.py

```diff
@@ -23,8 +23,7 @@
 # Reputation Service API Integration Testing
 
 import unittest
-import datetime
-import time
+from datetime import datetime, date
 import logging
 import pandas as pd
```

Comment:
Why do we need pandas and numpy in tests? I don't think they should be used in tests. Can we remove that?

Reply:
Unfortunately yes. I can probably fix that, but for some tests, we read dataframe from subfolders and use some nupy functions.